src/content/en/updates/2016/10/navigator.share.md

```diff
@@ -5,7 +5,7 @@ description: Sharing is caring. Chrome is running an Origin Trial to enable nati
 {# wf_published_on: 2016-10-19 #}
 {# wf_updated_on: 2016-10-19 #}
 
-# Introducing the Web Share API {: .page-title}
+# Introducing the Web Share API {: .page-title }
```

Comment:
Please do a pull before you make changes as I fixed on small issue here. It must be `{: .page-title }` including the spaces